### ISSUE TITLE
Unable to stop

### ISSUE THREAD

**Issue by mkasztelnik** [DarthSim/overmind#76]
I'm not able to stop overmind when [sidekiq](https://github.com/mperham/sidekiq) is defined in `Procfile`

```
jobs: bundle exec sidekiq -q computation -q mailers -q data_files -q notifications -q default
```

I have the following console output:

```
^Cjobs   | Interrupting...
jobs   | 2020-03-31T08:39:53.099Z pid=7029 tid=gq1pxscfd INFO: Shutting down
jobs   | 2020-03-31T08:39:53.099Z pid=7029 tid=gq1pxscfd INFO: Terminating quiet workers
jobs   | 2020-03-31T08:39:53.100Z pid=7029 tid=gq1qvod4l INFO: Scheduler exiting...
jobs   | 2020-03-31T08:39:53.200Z pid=7029 tid=gq1pxscfd INFO: Pausing to allow workers to finish...
jobs   | 2020-03-31T08:39:53.903Z pid=7029 tid=gq1pxscfd INFO: Bye!
jobs   | Killing...
```
It seems like the sidekiq is stopped (no process found in ps), but tmux session is not closed. After connecting to the jobs tmux session (`overminc c jobs`) `Pane is dead` message is shown.

## Versions
  * overmind with this issue 2.1.0 and 2.1.1 (OK in previous versions)
  * tmux 2.6
  * zsh 5.4.2 (x86_64-ubuntu-linux-gnu) 
  * asdf 0.7.6
  * sidekiq 6.0.5

**Comment by franzliedke** [comment]
I regularly encounter this issue as well.

If I can help you debug it, please let me know how. Unfortunately, I am not familiar with the inner workings of tmux. 

Thanks for this great tool, anyhow!!!

**Comment by agorf** [comment]
I wanted to report this issue as well. Did not happen with an older version, so doing `git bisect` should find the commit that broke things.

**Comment by mkasztelnik** [comment]
5bad403c0db05015b7b7e0bc2cf7a5cd40d699d5 is the first commit in which described error occurs.

**Comment by jamezrin** [comment]
Happens to me too, kinda annoying sometimes

**Comment by DarthSim** [comment]
Fixed this in 2.2.0
